deploy/helm_charts/envs/unsdg.yaml

```diff
@@ -56,13 +56,13 @@ nl:
 serviceGroups:
   recon: null
   svg:
-    replicas: 5
+    replicas: 3
   node:
-    replicas: 30
+    replicas: 3
   observation:
-    replicas: 30
+    replicas: 3
```

Comment:
A couple of thoughts:

- Slightly wonder if the LHS ratios (for observation esp) were there for a reason? Perhaps we rollout and monitor cpu/mem usage.
- Maybe link to the charts from your email in the description?
- @chejennifer also had run some load tests, and we can see about running it in staging or something?

Reply:
I added links to the monitoring dashboards & GKE resources page. So far memory usage looks good. the observation pods are using 10% of the total available memory https://screenshot.googleplex.com/9oRTw5nbvCjJsDT
@chejennifer what's the best way to run the load tests?

Reply:
this is the readme for the loadtester: https://github.com/datacommonsorg/website/tree/master/tools/nl/loadtest#nl-svg-load-tester

This tool was created to run against nodejs queries, but could update this URL to any URL to test: https://github.com/datacommonsorg/website/blob/master/tools/nl/loadtest/load.py#L33

Reply:
Maybe we could hit the fulfill and/or detect-and-fullfill endpoints since that's the primary load generator for UN site?

Reply:
Added tracking bug here: b/354275822